### PR TITLE
CCXDEV-4071 CCXDEV-4434 new chapter for restricted networks

### PR DESCRIPTION
**Pull request by xJustin**
https://issues.redhat.com/browse/CCXDEV-4071

https://issues.redhat.com/browse/CCXDEV-4434

this applies to Enterprise 4.8

https://deploy-preview-32558--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/remote-health-reporting-from-restricted-network.html

@quarckster this is ready for QA

### CODE REVIEW

**Comment by quarckster** [comment]
Thanks @xJustin lgtm.

File: modules/insights-operator-one-time-gather.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+* You are logged in to {product-title} as `cluster-admin`.
+
+.Procedure
+
+. Create a file named `gather-job.yaml` using this template:
++
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.8/docs/gather-job.yaml[]
```
Comment (jeana-redhat) [inline modules/insights-operator-one-time-gather.adoc:22]: Just want to check to see that you've gotten required approvals for this embedded file, as per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#embedding-an-external-file

**Review by jeana-redhat — COMMENTED** [review]
Very nice! Almost everything below is super minor formatting or style/L10N stuff.

File: modules/insights-operator-one-time-gather.adoc (hunk 1 above)
Comment (xJustin) [reply modules/insights-operator-one-time-gather.adoc:22]: Yes I have. It was not my first choice, but this template updates regularly at the moment. If it ever solidifies, I hope to just embed it into the doc.

File: modules/insights-operator-enable-obfuscation.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+You can enable obfuscation to mask sensitive and identifiable IPv4 addresses and cluster base domains Insights Operator sends to link:https://cloud.redhat.com[cloud.redhat.com].
+
+[WARNING]
+====
+Although this feature is available, Red Hat recommends keeping obfuscation off for a more effective support experience.
+====
+
+Obfuscation assigns non-identifying values to cluster IPv4 addresses, and uses a translation table retained in memory to change IP addresses to their obfuscated versions throughout the Insights Operator archive before uploading the data to link:https://cloud.redhat.com[cloud.redhat.com].
+
+For cluster base domains, obfuscation changes the base domain to a hardcoded substring. For example, `cluster-api.openshift.example.com` becomes `cluster-api.<CLUSTER_BASE_DOMAIN>`.
```
[the remark this answers is not in the document]
Comment (xJustin) [reply modules/insights-operator-enable-obfuscation.adoc:19]: Yes, obfuscation changes all base domains to that same value in the archive.

File: modules/insights-operator-enable-obfuscation.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as `cluster-admin`.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Search for the *support* secret using the *Search by name...* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
+. Click the *Options* menu {kebab} -> *Edit Secret*.
```
[the remark this answers is not in the document]
Comment (xJustin) [reply modules/insights-operator-enable-obfuscation.adoc:29]: Good call, I changed them.

**Comment by xJustin** [comment]
@jeana-redhat thanks for all the tips! I have made the changes. Is there anything else I need to do to get this merged?

File: modules/insights-operator-enable-obfuscation.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+For cluster base domains, obfuscation changes the base domain to a hardcoded substring. For example, `cluster-api.openshift.example.com` becomes `cluster-api.<CLUSTER_BASE_DOMAIN>`.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as `cluster-admin`.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Search for the *support* secret using the *Search by name...* field. If it does not exist, click *Create* -> *Key/value secret* to create it.
```
Comment (jeana-redhat) [inline modules/insights-operator-enable-obfuscation.adoc:28]: I apologize, I missed this one on my first run through.
s/Search by name.../Search by name

**Review by jeana-redhat — COMMENTED** [review]
Just found one small thing I missed before. This looks great!

File: modules/insights-operator-enable-obfuscation.adoc (hunk 4 above)
Comment (xJustin) [reply modules/insights-operator-enable-obfuscation.adoc]: Changed it, I wasn't sure because the field has the three dots in it.

Comment (jeana-redhat) [reply modules/insights-operator-enable-obfuscation.adoc]: Yeah, I totally understand where the inclination comes from :) Per style guide: 

> In documentation, do not include punctuation in a label, such as an ellipsis that indicates that more information is required from the user, unless one of the following conditions applies:
> - The omission of the punctuation would result in confusion over the meaning of the interface element.
> - You are using a tool that extracts the wording on the label, including the punctuation, for use in the documentation.

**Pull request merged**